### PR TITLE
Preserve original error during SCC Reconcile

### LINKED ISSUE #14550: oc adm policy reconcile-sccs: improve error reporting when user doesn't have permissions to a namespace
When I'm trying to run `oc adm policy reconcile-sccs` command under unprivileged user I've got an error `openshift-infra is not a valid namespace` that confuses me as a user and doesn't tell me the real cause.

```console
$ oc whoami
developer
$ oc adm policy reconcile-sccs 
error: openshift-infra is not a valid namespace
See 'oc adm policy reconcile-sccs -h' for help and examples.
$ oc adm policy reconcile-sccs --loglevel=9
I0609 17:27:48.859217    4478 loader.go:354] Config loaded from file /home/coder/.kube/config
I0609 17:27:48.859967    4478 cached_discovery.go:118] returning cached discovery info from /home/coder/.kube/127.0.0.1_8443/servergroups.json
I0609 17:27:48.860307    4478 round_trippers.go:386] curl -k -v -XGET  -H "Accept: application/json, */*" -H "User-Agent: oc/v1.6.1+5115d708d7 (linux/amd64) kubernetes/010d313" -H "Authorization: Bearer qvk2g1-Rj7-7hZ4BFiC3tC3sPobfASrTOWG_VB6L3t4" https://127.0.0.1:8443/api/v1/namespaces/openshift-infra
I0609 17:27:48.881430    4478 round_trippers.go:405] GET https://127.0.0.1:8443/api/v1/namespaces/openshift-infra 403 Forbidden in 21 milliseconds
I0609 17:27:48.881450    4478 round_trippers.go:411] Response Headers:
I0609 17:27:48.881456    4478 round_trippers.go:414]     Date: Fri, 09 Jun 2017 15:27:48 GMT
I0609 17:27:48.881460    4478 round_trippers.go:414]     Cache-Control: no-store
I0609 17:27:48.881465    4478 round_trippers.go:414]     Content-Type: application/json
I0609 17:27:48.881472    4478 round_trippers.go:414]     Content-Length: 290
I0609 17:27:48.881496    4478 request.go:991] Response Body: {
  "kind": "Status",
  "apiVersion": "v1",
  "metadata": {},
  "status": "Failure",
  "message": "User \"developer\" cannot get namespaces in project \"openshift-infra\"",
  "reason": "Forbidden",
  "details": {
    "name": "openshift-infra",
    "kind": "namespaces"
  },
  "code": 403
}
F0609 17:27:48.881634    4478 helpers.go:119] error: openshift-infra is not a valid namespace
See 'oc adm policy reconcile-sccs -h' for help and examples.
```

##### Version
```console
$ oc version
oc v3.6.0-alpha.2+c824f5c-17-dirty
kubernetes v1.6.1+5115d708d7
features: Basic-Auth

Server https://127.0.0.1:8443
openshift v3.6.0-alpha.1+5357892-777
kubernetes v1.6.1+5115d708d7
```

##### Steps To Reproduce
1. `oc cluster up`
2. `oc adm policy reconcile-sccs`

##### Current Result
`error: openshift-infra is not a valid namespace` error is shown.

##### Expected Result
`User "developer" cannot get namespaces in project "openshift-infra"` error should be shown.

### PR DESCRIPTION
**Pull request by adelton**
Fixes #14550

### CODE REVIEW

**Comment by adelton** [comment]
Addressing https://github.com/openshift/origin/issues/14550.

**Comment by enj** [comment]
While this is pretty high up in the stack and thus it is easy to locate the source of the error, something like this is more useful:

`return fmt.Errorf("Failed to GET reconcile SCC namespace %s: %v", o.InfraNamespace, err)`

Please update your commit message to be useful to someone without context of the original issue.

File: pkg/cmd/admin/policy/reconcile_sccs.go [hunk 1]
```diff
@@ -130,7 +130,7 @@ func (o *ReconcileSCCOptions) Validate() error {
 		return errors.New("a SCC client is required")
 	}
 	if _, err := o.NSClient.Get(o.InfraNamespace, metav1.GetOptions{}); err != nil {
-		return fmt.Errorf("%s is not a valid namespace", o.InfraNamespace)
+		return fmt.Errorf("Failed to GET reconcile SCC namespace %s: %v", o.InfraNamespace, err)
```
Comment (php-coder) [inline pkg/cmd/admin/policy/reconcile_sccs.go]: Neat: I'd rather use `%q` specifier that will quote the string.

Comment (php-coder) [inline pkg/cmd/admin/policy/reconcile_sccs.go]: P.S. It's fully optional and up to you. It doesn't block merging of this.

**Comment by enj** [comment]
[test]

**Comment by enj** [comment]
[merge][severity:bug]

**Comment by adelton** [comment]
So, that continuous-integration/openshift-jenkins/merge thing failed with

```
Network flake

Unable to connect to a remote server
Indication 1
	
Job Stage Failed

Stage RUN EXTENDED TESTS [00h 35m 50s] failed.
Indication 2
	
Extended test failure

An extended test failed.
Indication 3 Indication 4 
```

What's the next plan / steps for getting this oneliner merged?

**Comment by enj** [comment]
@adelton congrats!

**Pull request merged**